src/components/TitleCardList.tsx

```diff
@@ -152,52 +40,171 @@ const TitleCardList: React.FC<IProps> = ({
   icon,
   isCode,
 }) => (
-  <Table className={className}>
-    <TableHeader>
-      {icon && <StyledIcon name={icon} />}
+  <Box
+    bg="background"
+    boxShadow="tableBoxShadow"
+    width="100%"
+    margin="2rem"
+    border-radius="2px"
```

Comment:
https://chakra-ui.com/docs/styled-system/theme#border-radius
```suggestion
    border-radius="sm"
```